src/company_cmd.cpp

```diff
@@ -389,7 +389,7 @@ set_name:;
 			SetDParam(1, STR_NEWS_COMPANY_LAUNCH_DESCRIPTION);
 			SetDParamStr(2, cni->company_name);
 			SetDParam(3, t->index);
-			AddNewsItem(STR_MESSAGE_NEWS_FORMAT, NT_COMPANY_INFO, NF_COMPANY, NR_TILE, c->last_build_coordinate, NR_NONE, UINT32_MAX, cni);
+			AddNewsItem(STR_MESSAGE_NEWS_FORMAT, NT_COMPANY_INFO, NF_COMPANY, NR_TILE, static_cast<uint32_t>(c->last_build_coordinate), NR_NONE, UINT32_MAX, cni);
```

Comment:
I investigated overloading this to pass a TileIndex directly, and found we already have one! Try `AddTileNewsItem()` instead? (I see this PR adds a cast there, but one less cast is one less cast!)

Reply:
Yes and no .. the `AddTileNewsItem` sets `NF_NO_TRANSPARENT | NF_SHADE | NF_THIN` while we want `NF_COMPANY`. So you can't actually use that function without loss of functionality :( And I didn't want to make more changes than needed in this PR.